scripts/README.md

```diff
@@ -0,0 +1,30 @@
+# Scripts
+
+## Indy-SDK
+
+### Debian/Ubuntu
+
+```bash
+./scripts/debian-libindy/install-indy.sh
```

Comment:
Should we now guide to use `install-indy.sh` script in the main `README.md` as well? **This comment is not for this file but `../README.md`**

Reply:
Updated the step in README but left the indy-sdk-link there anyway, if it is needed e.g. with other linux distros.